scripts/execute_ebpf_cicd_tests.ps1

```diff
@@ -33,12 +33,17 @@ foreach ($VM in $VMList) {
         -RestartExtension $RestartExtension
 }
 
-# Run XDP Tests.
-Invoke-XDPTestsOnVM $Config.Interfaces $VMList[0].Name
+if ($RunKmStressTests -eq $false) {
```

Comment:
why $false instead of $true?

Reply:
The km stress tests are scheduled runs and I've extended this (existing) script to handle those tests as well.  I'd previously added a separate script for the km stress tests, but a prior feedback recommended that we do this instead.

The `$RunKmStressTests` parameter ensures that when set to true, _only_ the (new) stress tests are run. The existing tests, i.e, `Invoke-XDPTestsOnVM`, `Invoke-ConnectRedirectTestsOnVM` and `Invoke-ConnectRedirectTestsOnVM` are already handled by other jobs, so re-executing them again along with the stress tests would be redundant.

Reply:
The name `RunKmStressTests` does not imply that "only" the stress tests are run, it implies that the stress tests are run in addition.  Either change the name or change the object.  E.g., if it _disables_ running non-stress tests, then it should be phrased as such.

Reply:
Fixed. Renamed `RunKmStressTests` to `RunKmStressTestsOnly`.